docker-for-windows/faqs.md

```diff
@@ -117,9 +117,14 @@ Yes, you can run VirtualBox along with Docker Desktop if you have enabled the [
 
 ## Windows requirements
 
+### Can I run Docker Desktop on Windows Server?
+
+No, running Docker Desktop on Windows Server is not supported.
+
 ### How do I run Windows containers on Docker Desktop on Windows Server?
```

Comment:
Do we want to change this as well?

Reply:
Could you provide more info? Do you mean updating the heading or combining both questions?

Reply:
Oh sorry, I meant the heading looks wrong as it has Docker Desktop in it, maybe it should just be `How do I run Windows containers on Windows Server?`?

Reply:
Aah, yes. That makes sense.

Reply:
Updated